index.html

```diff
@@ -11,93 +10,101 @@
+          <li class="navlink"><a href="#" class="navlink">About</a></li>
+          <li class="navlink"><a href="#" class="navlink">Contact</a></li>
+          <li class="navlink"><a href="#" class="navlink"><img src="./img/Icon - Mail.png" alt="email"></a></li>
+        </ul>
+      </nav>
+    </header>
   <!-- Home -->
-  <section>
-    <img class="ellipse18" src="./img/Ellipse 18.png">
+  <section class="hero-container">
     <div class="home-content">
-      <h1>Maher Ramadan</h1>
+      <h1 class="hero-title">Maher Ramadan</h1>
     </div>
     <div class="home-text">
       <h2>Software Developer</h2>
     </div>
     <div class="paragraph">
-      <p>I can help you build a product,
+      <p class="hero-description">I can help you build a product,
         feature or website Look through
         some of my work and experience! If you like what you see
         and have a project that needs coding,don’t
         hesitate to contact me. </p>
     </div>
-  </section>
-  <hr>
-  <!-- line1 -->
-  <section>
-    <div class="list-technologies">
-      <div class="ellipse19"></div>
-      <div class="languages">
-        <h4>Languages</h4>
-        <ul class="menu-language">
-          <li>JavaScript</li>
-          <li>Ruby</li>
-          <li>Html</li>
-          <li>CSS</li>
-        </ul>
-      </div>
-      <div class="Frameworks">
-        <h4>Frameworks</h4>
-        <ul class="menu-frameworks">
-          <li>Bootstrap</li>
-          <li>Ruby on Rails</li>
-          <li>RSPec</li>
-          <li>CapyBara</li>
-          <li>Selenium</li>
-        </ul>
-      </div>
-      <div class="Skills">
-        <h4>Skills</h4>
-        <ul class="menu-skills">
-          <li>Codekit</li>
-          <li>Github</li>
-          <li>Codepen</li>
-          <li>Gitlab</li>
-          <li>Terminal</li>
-        </ul>
+    <hr class="line-01">
+    <!-- line1 -->
+    <section>
+      <div class="list-technologies">
+        <!-- <div class="ellipse19"></div> -->
+        <div class="languages">
+          <h4>Languages</h4>
+          <ul class="menu-language">
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <div class="Frameworks">
+          <h4>Frameworks</h4>
+          <ul class="menu-frameworks">
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="Skills">
+          <h4>Skills</h4>
+          <ul class="menu-skills">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+        <div class="Skills Skills-empty"></div>
       </div>
-      <div class="Skills"></div>
-    </div>
+    </section>
   </section>
```

Comment:
- [ Optional ] For desktop version of your portfolio Kindly work on the toolbar section (In the desktop version there is no hamburger menu) and kindly adjust the font size and alignment to center  of your "list-technologies" Just like the figma design for Good UI.

Your project Look likes this : For desktop version 
![Capture](https://user-images.githubusercontent.com/93598189/200417953-cc3beade-7d4e-4c87-90e8-c93f93132477.PNG)

The Figma design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/200418155-29b09c96-e205-474e-a2ca-a325f66bca55.PNG)

Reply:
ok thank you so much , i will fix it

Reply:
why i could see it like this , i mean with burger menu when its mobile version.

Reply:
![imagen](https://user-images.githubusercontent.com/90804443/200420691-e441d901-2cc6-4406-af79-84de5b6d40ec.png)

Reply:
![imagen](https://user-images.githubusercontent.com/90804443/200420756-df5f5138-71dd-4501-a10a-6ce38407f2a2.png)

right now its desktop version and the menu is like this

Reply:
Okay @maherramadan78 I see that but please update the changes on your deploy project.